examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -76,6 +76,33 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion forEach-over-func
   };
 
+  {
+    var sequence = [];
+    // #docregion list-from-1
+    var copy1 = sequence.toList();
+    var copy2 = new List.from(sequence);
+    // #enddocregion list-from-1
+  }
+
+  {
+    var sequence = [];
+    // #docregion list-from-2
+    var sequence = [1, 2, 3]; // Creates a List<int>.
```

Comment:
You have two `var sequence` declarations.

Reply:
Done.